.github/workflows/core_code_review_weu.yaml

```
@@ -0,0 +1,66 @@
+name: WEU - Core TF Validation (PROD)
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths:
+      - "src/core/*"
+      - "src/psql/*"
+      - ".devops/core-code-review-pipelines.yml"
```

If you agree, I would change it with the path of this action to trigger it on change
```suggestion
      - ".github/workflows/core_code_review_weu.yaml"
```

---

I agree